webpack.config.js

```diff
@@ -84,7 +84,23 @@ module.exports = (env, argv) => {
                         {
                             loader: 'style-loader',
                             options: {
-                                insertAt: 'top'
+                                insert: function insertAtTop(element) {
+                                    var parent = document.querySelector('head');
+                                    // eslint-disable-next-line no-underscore-dangle
+                                    var lastInsertedElement =
+                                        window._lastElementInsertedByStyleLoader;
+
+                                    if (!lastInsertedElement) {
+                                        parent.insertBefore(element, parent.firstChild);
+                                    } else if (lastInsertedElement.nextSibling) {
```

Comment:
is there a possibility that `lastInsertedElement` is not null but does not have a `nextSibling` attribute.

Reply:
nvm if its a valid html element then it will have nextSibling.

Reply:
I don't see how unless something else on the page hijacks `window._lastElementInsertedByStyleLoader` - but even then we'll fall back on `parent.appendChild(element)`, which will have a reasonable end result regardless.

Of course if someone else does use `window._lastElementInsertedByStyleLoader` they could break this in multiple ways, but I see that as an acceptable risk.